Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1126 @@
+        4: entity.name.type.textile
+        5: entity.name.tag.codeblock.textile
+      push: block-bc-body
+
+  extended-block-bc-body:
+    - meta_scope: markup.raw.textile
+    - include: next-block-pop
+
+  block-bc-body:
+    - meta_scope: markup.raw.textile
+    - include: empty-line-pop
+
+###[ BLOCKS / BLOCK QUOTES ]###################################################
+
+  # https://textile-lang.com/doc/block-quotations
+
+  quotation-blocks:
+    - match: |-
+        (?x:
+          ^
+          (bq)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.blockquote.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.blockquote.textile
+      push: extended-block-bq-body
+    - match: |-
+        (?x:
+          ^
+          (bq)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.blockquote.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.blockquote.textile
+      push: block-bq-body
+
+  extended-block-bq-body:
+    - meta_scope: markup.quote.textile
+    - include: next-block-pop
+    - include: inlines
+    - include: html
+
+  block-bq-body:
+    - meta_scope: markup.quote.textile
+    - include: empty-line-pop
+    - include: inlines
+    - include: html
+
+###[ BLOCKS / FOOTNOTES ]######################################################
+
+  # https://textile-lang.com/doc/footnotes
+
+  footnotes:
+    - match: |-
+        (?x:
+          ^
+          (fn\d+)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.)
+        )
       captures:
         1: entity.name.tag.footnote.textile
-        3: entity.name.type.textile
-        4: entity.name.tag.footnote.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.footnote.textile
       push:
         - meta_scope: markup.other.footnote.textile
-        - match: ^$
-          captures:
-            1: entity.name.tag.footnote.textile
-            3: entity.name.type.textile
-            4: entity.name.tag.footnote.textile
-          pop: true
-        - include: inline
-        - include: scope:text.html.basic
-    - match: '(^table([<>=()]+)?)(\([^)]*\)|\{[^}]*\})*(\.)'
+        - include: empty-line-pop
+        - include: inlines
+        - include: html
```

Comment:
Block bodies may be worth named contexts for extending syntaxes to individually add patterns.

```
      ...
      push: footnote-body

  footnote-body:
    - meta_scope: markup.other.footnote.textile
    - include: empty-line-pop
    - include: inlines
    - include: html
```